code/common/pcapdecoder.q

```diff
@@ -0,0 +1,87 @@
+\d .pcap
+
+// size of headers in bytes and dict of protocol code conversions
+globheader: 24;
+packetheader: 16;
+allcodes:(enlist 6)!(enlist `TCP);
```

Comment:
Can you include a link to the PCAP spec here as an explanation for these values?

On a side note, I assume in theory these values could be different for different versions of the PCAP spec (e.g. some future version of the spec could change these), so might be good to specify the version this is written for (maybe even have the code check the version, there's probably some header in the file with that info?)